### PR TITLE
Updates for Azure

### PR DESCRIPTION
**Pull request by ctro**
Mostly README and a new Azure kitchen example

### CODE REVIEW

File: arm/cookbooks/motd/README.md [hunk 1]
```diff
@@ -81,3 +81,36 @@ kitchen test
...
+```
+
+0. `kitchen test`.  If this fails trying to delete non-existing resources, `rm -rf .kitchen` and try again.
+
+Benchmarks:
+
+| VM | Cores | Memory | t1 | t2 | t3 |
+| -- | -- | -- | -- | -- | -- |
+| Standard_D1 | 1 | 3.5G | 9:00 | 9:15 | 7:35 |
+| Standard_D2 | 2 | 7G | 8:01 | 9:53 | 8:04 |
```
Comment (ctro) [inline arm/cookbooks/motd/README.md]: I'm not really sure what these tell us...

File: arm/cookbooks/motd/.kitchen.local.azure.example.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      name: winrm
+
+suites:
+  - name: default
+    run_list:
+      - recipe[motd::default]
+    verifier:
+      inspec_tests:
+        - test/recipes
+        - spec/unit/recipes/default_spec.rb
```
Comment (pburkholder) [inline arm/cookbooks/motd/.kitchen.local.azure.example.yml]: Ah -- this is any easy mistake to make. The tests in `spec/unit/...` don't have anything to do with test-kitchen. They are only for use with ChefSpec (think RSpec) unit tests while the stuff in `test/recipes` are the kitchen integration tests (think Selenium).  

So the test for Chef cookbooks are:
1. lexical/style (rubocop, foodcritic)
2. unit tests (rspec w/ ChefSpec)
3. integration tests (test-kitchen)

In short, just delete line 26....

File: arm/cookbooks/motd/README.md (hunk 1 above)
Comment (pburkholder) [inline arm/cookbooks/motd/README.md]: I was keeping some notes on long these tests take to run. After the zippiness of running kitchen w/ Docker the 8min wait for win2012r2 to spin up, converge, validate, and destroy itself feels interminable. Whether there's any remedy for that is unclear. How about change line 111ff to:

```
Benchmarks:
 
Full integration tests against Win2012R2 take a _long_ time. Here's what we observed in minutes:seconds for various instance types:

| VM | Cores | Memory | test1 mm:ss | test2 mm:ss | test3 mm:ss |
| -- | -- | -- | -- | -- | -- |
| AzureSM Standard_D1 | 1 | 3.5G | 9:00 | 9:15 | 7:35 |
| AzureSM Standard_D2 | 2 | 7G | 8:01 | 9:53 | 8:04 |
```

**Comment by pburkholder** [comment]
left a couple notes...

**Comment by ctro** [comment]
OK, I made those 2 small changes, thanks for the feedback!

**Comment by ctro** [comment]
FYI, Just rebased this on the updated `master`
